routers/likes.py

```
@@ -1,31 +1,143 @@
+        posts[post_index]["like_count"] = posts[post_index].get("like_count", 0) + delta
+        write_json(settings.posts_file, posts)
+
+
+@router.get("/posts/liked", response_model=ListPostILiked)
+def get_posts_liked(user_id: CurrentUserId, page: Page = 1):
+    """내가 좋아요한 게시글 목록"""
+    likes = read_json(settings.likes_file)
+    posts = read_json(settings.posts_file)
+
+    # 내가 좋아요한 post_id 목록 (최신순 정렬)
+    my_likes = [l for l in likes if l["user_id"] == user_id]
```

Ruff (0.14.13)

Ambiguous variable name: `l`

(E741)